src/api/lib/tasks/statistics/github/code_frequency.rake

```diff
@@ -0,0 +1,30 @@
+require 'net/http'
+
+namespace :statistics do
+  namespace :github do
+    desc 'Exports the number of additions and deletions per week to a file'
+    task code_frequency: :environment do
+      uri = URI('https://api.github.com/repos/openSUSE/open-build-service/stats/code_frequency')
+      response = Net::HTTP.get_response(uri)
+
+      # The first time you make this request to github, it returns an empty json response with status 202
+      # because github generates the statistics asynchronously. Once they are generated will be 200.
+      if response.code == '202'
+        puts 'Statistics are being generated in the background by github. Please re-run this task in a minute to get the results.'
```

Comment:
Yeah I did consider doing that but decided against it because there is no guarantee that the stats will be generated in 5 secs, and also because this task will not be used very often, its really just for Ana's presentation, so it didn't seem worth putting any more effort into this IMO.